e2e/cypress/integration/ui/settings/04_team_details.spec.ts

```diff
@@ -0,0 +1,231 @@
+describe('team management', () => {
+  let adminToken = '';
+  const now = Cypress.moment().format('MMDDYYhhmm');
+  const cypressPrefix = 'cypress-test';
+  const teamName = `${cypressPrefix} team ${now}`;
+  const teamID = `${cypressPrefix}-testing-team-custom-id-${now}`;
+  const project1ID = `${cypressPrefix}-project1-${now}`;
+  const project1Name = `${cypressPrefix} project1 ${now}`;
+  const project2ID = `${cypressPrefix}-project2-${now}`;
+  const project2Name = `${cypressPrefix} project2 ${now}`;
+  const unassigned = '(unassigned)';
+  let teamUIRouteIdentifier = '';
+  const nameForUser = cypressPrefix + ' user ' + now;
+  const usernameForUser = cypressPrefix + 'testing-user-' + now;
+  let userMembershipID = '';
+
+  let iamVersion = <string><Object>Cypress.env('IAM_VERSION');
+  // assume 2.0 if not in CI. if you wish something different start cypress with
+  // IAM_VERSION set to what you are testing.
+  if (iamVersion === undefined) {
+    iamVersion = 'v2.0';
+  }
+
+  const describeIAMV2 = iamVersion.match(/v2/) ? describe : describe.skip;
+  const describeProjectsEnabled = iamVersion === 'v2.1' ? describe : describe.skip;
```

Comment:
I think I've seen this before? If I'm not wrong, can we promote into a shared helper function?